CHANGELOG.md

```
@@ -2,6 +2,10 @@
 
 All notable changes to this project will be documented in this file.
 
+### Features
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)